packages/react/src/components/MDXComponents/overview.mdx

```diff
@@ -0,0 +1,64 @@
+<Meta title="MDX Components/MDX Documentation Guidelines" />
+# MDX Documentation Guidelines  
+
+This document outlines the best practices and rules for writing MDX documentation 
+for components under `@carbon-labs`. 
+
+Following these guidelines will ensure all documented components have the same styles
+as the ones found in the [Carbon Design System](carbondesignsystem.com) page.
+
+
+## Directory Structure  
+- All `.mdx` files **must** be placed under:  
+  ```
+  src/components/ComponentName/docs/
+  ```
+- The `Meta` component must be included at the top of the file with a `title` in the format:  
```

Comment:
```suggestion
- The `Meta` Storybook MDX component must be included at the top of the file with a `title`. This allows you to determine where the file is included in the side navigation, instead of it being placed based on directory path by default.  
```